x-pack/plugins/alerts/server/types.ts

```diff
@@ -86,6 +86,7 @@ export interface AlertType {
     state?: ActionVariable[];
     params?: ActionVariable[];
   };
+  errorCB?: ({ alertId, message }: { alertId: string; message: string }) => Promise<void>;
```

Comment:
nit: `onError` seems like a more standard callback name to me, I've not seen much of the `*CB` form.